placeholder-material/src/main/java/com/google/accompanist/placeholder/material/Placeholder.kt

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.accompanist.placeholder.material
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.contentColorFor
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.compositeOver
+import com.google.accompanist.placeholder.PlaceholderHighlight
+import com.google.accompanist.placeholder.placeholder
+
+/**
+ * Contains default values used by [Modifier.placeholder] and [PlaceholderHighlight].
+ */
+object PlaceholderDefaults {
+    /**
+     * Returns the value used as the the `color` parameter value on [Modifier.placeholder].
+     *
+     * @param backgroundColor The current background color of the layout. Defaults to
+     * `MaterialTheme.colors.surface`.
+     */
+    @Composable
+    fun color(
+        backgroundColor: Color = MaterialTheme.colors.surface
```

Comment:
I'm not sure about accepting a `backgroundColor` param — do you think it would be common to supply this? Also the alpha mod is hard coded.

What about keeping this as a `color` param like the non-material version but defaulting this to `LocalContentColor` alpha mod-ed and compositited over `surface`? That would keep all the defaulted/assumed material colors in default params so it's easy

What about accepting 3 params for `backgroundColor`, `contentColor` and `contentColorAlpha` (defaulted to `surface`, `LocalContentColor` and `0.1f`) to make this very flexible and easy to tweak?

Reply:
Done.